src/Installer/ReleaseNotes.md

```diff
@@ -1,6 +1,8 @@
 ## DEV_VERSION_NUMBER: DEV_RELEASE_DATE
 * Chorus: Upgrade Mercurial to version 3
-## 2.2.14 23/Jan/2015
+## 2.2.15 4/May/2015
+* Chorus: Fix overflow error when receiving very large project data with resumable option
+## 2.2.14 3/Feb/2015
```

Comment:
Is the Jan or Feb date right for 2.2.14? Seems odd to reset it at this late date.